app/assets/stylesheets/layout/_layout.scss

```diff
@@ -45,6 +45,18 @@ body {
       margin-bottom: 40px;
     }
   }
+
+}
+
+.registrations .box,
+.sessions .box {
+  margin: 0 auto;
+  width: 40%;
+  
+  input[type="submit"] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3